docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -525,6 +525,123 @@ as follows:
+the parameter should be modeled as follows:
+
+.. code-block:: Python
+
+	from scipy.stats import norm
+
+	# mean and 0.975-quantile of normal distribution for mean difference (MD)
+	mean = 15.1
+	q_975 = 24.2
+
+	# 0.975-quantile of standard normal distribution (=1.96, approximately)
+	q_975_stdnorm = norm().ppf(0.975)
+
+	std = (q_975 - mean) / q_975_stdnorm # std dev of normal distribution
+
+	# Frozen normal distribution for MD, representing uncertainty in our effect size
+	bw_md_distribution = norm(mean, std)
+
+	# random sample from effect size distribution
+	bw_md_per_10_mg_iron = bw_md_distribution.rvs()
+
+.. note::
+
+	The Haider et al. 2013 paper did not report if the dose of iron was 
+	measured as elemental iron or as an iron compound such as NaFeEDTA. We 
+	operated under the assumption that 10 mg of daily iron consumption, as 
+	referenced in the Haider paper, represented 10 mg of *elemental* iron.
+
+Therefore, we investigated the amount of daily iron consumption that a 
+pregnant mother would likely consume through iron fortification of staple 
+foods in our locations of interest in order to scale the effect size 
+accordingly. See the table below:
+
+.. list-table:: Daily Iron Consumption Parameters
+  :widths: 5 5 5ight
+  :header-rows: 1
+
+  * - Location
+    - Concentration of forticant in flour
+    - Amount of fortifiable flour consumed daily
+  * - Ethiopia
+    - 30 mg NaFeEDTA / kg flour  [1]
+    - 100 g (IQR: 77.5, 200) [2]
+  * - India
+    - 14 to 21.25 mg NaFeEDTA / kg flour [1]; assume midpoint value of **17.6**
+    - 100 g (IQR: 77.5, 200) [2*]
+  * - Nigeria
+    - 40 mg NaFeEDTA / kg flour [1]
+    - 100 g (IQR:77.5, 200) [2*]
+
+.. note::
+
+	While there was data for the concentration of iron forticants other than 
+	NaFeEDTA for the locations in this table, we conservatively chose to  use 
+	the concentrations of NaFeEDTA because elemental iron is uesd at a *lower* 
+	concentration in flour when NaFeEDTA is used than for other forticants.
+
+[1] https://tinyurl.com/wka9mgh
+
+[2] https://www.ephi.gov.et/images/pictures/National%20Food%20Consumption%20Survey%20Report_Ethiopia.pdf (table 18; women)
+
+* In the absence of better data, we assumed that individuals in Nigeria and 
+India consumed the same amount of fortifiable flour per day as individuals in 
+Ethiopia.
+
+.. todo:: 
+
+	Find better data and replace values for flour consumption per day for 
+	Nigeria and India.
+
+The amount of elemental iron consumed daily, in miligrams per person, (at the 
+draw level) should be calculated as such:
+
+.. math::
+
+	\frac{\text{X mg NaFeEDTA}}{\text{kg flour}} * \frac{\text{Y g flour consumed daily}}{\text{person}} * \frac{\text{1 kg flour}}{\text{1,000 g flour}} * \frac{\text{55.845 g elemental iron}}{\text{367.047 g NaFeEDTA}} = \frac{\text{Z mg elemental iron consumed daily}}{\text{person}}
+
+.. code-block:: Python
+
+	## WRITE CODE THAT SAMPLES FROM AMOUNT OF FLOUR CONSUMED PER DAY (BETA DISTRIBUTION??)
```

Comment:
I will think about an appropriate distribution to use here. Do we have some idea of what should be the _maximum_ amount of daily flour consumed?

Reply:
We only have 25th, 50th, and 75th percentiles.

Reply:
The situation here is similar to what I wrote above about our data for fortification levels in India: This data is telling us about the heterogeneity between households in Ethiopia.

In this case, however, it is unclear to me how to pull out a single parameter to use for the model. E.g. do we want the mean or the median? And if we want the mean, how do we estimate it?

Therefore, I think we should use the data to directly estimate the distribution of flour consumption among the population, i.e. to model individual heterogeneity rather than just using a single parameter with uncertainty.

The simplest way I can think of to estimate the distribution is to use a mixture of uniform distributions supported between the quartiles (this is also the [maximum entropy](https://en.wikipedia.org/wiki/Maximum_entropy_probability_distribution) strategy for choosing the least informative distribution subject to the constraints). That is, for simulants who eat fortified flour, assign their daily wheat flour consumption using the following procedure:

- Sample 25% of the simulants uniformly between 0 and Q1=77.5
- Sample 25% of the simulants uniformly between Q1=77.5 and Q2=100
- Sample 25% of the simulants uniformly between Q2=100 and Q3=200
- Sample 25% of the simulants to have consumption >Q3=200. Either we can choose some maximum (maybe Q4=500 g?) and sample uniformly on [Q3,Q4], or we can choose some distribution supported on [Q3, infinity) and sample from that (e.g. a shifted [exponential distribution](https://en.wikipedia.org/wiki/Exponential_distribution) or [Pareto distribution](https://en.wikipedia.org/wiki/Pareto_distribution), which are also maximum entropy distributions subject to certain constraints).

Reply:
Ok. Notably, the reported figure was the amount of fortifiable wheat eaten among those who reported that they ate fortifiable wheat, so the lower bound should be > 0. What if we took the minimum 25th percentile and maximum of the 75th percentile and used those as the upper and lower bounds? 
![image](https://user-images.githubusercontent.com/54918472/77583645-aaf08900-6e9e-11ea-9376-504a70647ad0.png)

So...
Sample 25% of the simulants uniformly between **50** and Q1=77.5
Sample 25% of the simulants uniformly between Q1=77.5 and Q2=100
Sample 25% of the simulants uniformly between Q2=100 and Q3=200
Sample 25% of the simulants uniformly between Q3=200 and **350.5**

Obviously we will be truncating a little bit here, but at least we will capture some regional variation?

Reply:
In the absence of more data, I don't necessarily agree that the lower bound should be >0, because increasing it makes our estimates less conservative.  Since the uniform distribution assigns probability 0 to the value 0 (or any other value), a lower bound of 0 is consistent with the data being from people who eat fortifiable wheat. It just says that there's some small probability that they eat, say <1g per day.

If we increase the lower bound to 50, we know we will be overestimating some people's wheat flour consumption (e.g. 25% of people in Amhara), so I think it is better to go with the conservative lower bound of 0.

For the upper bound, it seems fine to go with the maximum 75th percentile. We know we will be underestimating some people's wheat consumption, but that makes this a conservative estimate.

Reply:
Ok, yeah that sounds good to me. Will update!